tests/Integration/Behaviour/Features/Scenario/Shipment/shipment.feature

```diff
@@ -28,7 +28,7 @@ Feature: Retrieving shipment for orders
   Scenario: Retrieve shipmets for existing order
     Given the order "bo_order1" should have the following shipments:
       | shipment  | carrier         | tracking_number | address | shipping_cost_tax_excl | shipping_cost_tax_incl |
-      | shipment1 | default_carrier |                 | US      |                    7.0 |                   7.42 |
+      | shipment1 | default_carrier |                 | US      |                    0.0 |                   0.0 |
```

Comment:
Why does the shipment suddenly becomes zero? Doesn't it mean there's a bug compared to previously?

Reply:
Because when we give null for carrierId params for validateOrder, we have 0 for shipping_cost inside order

Reply:
Never mind, it's solved.